UnitTestFrameworkPkg/ReadMe.md

```diff
@@ -1366,6 +1366,10 @@ After that, the following commands will set up the build and run the host-based
 # stuart_setup -c ./.pytool/CISettings.py TOOL_CHAIN_TAG=<GCC5, VS2019, etc.>
 stuart_setup -c ./.pytool/CISettings.py TOOL_CHAIN_TAG=VS2019
 
+# Mu specific step to clone mu repos required for ci check
+# stuart_ci_setup -c ./.pytool/CISettings.py TOOL_CHAIN_TAG=<GCC5, VS2019, etc.>
+stuart_ci_setup -c ./.pytool/CISettings.py TOOL_CHAIN_TAG=VS2019

```

Comment:
I think all the other changes except this one can be dropped. They are not changing anything of value. It claims to be changing the readme to what the actual codebase has, but they are all examples that are changing.